sass/components/_asset-card.scss

```diff
@@ -2,12 +2,13 @@ $asset-card-padding: 0.4rem;
 
 .asset-card {
     display: grid;
+    max-width: 94vw;

```

Comment:
Why is this needed? 94vw seems to be "approximately" the width of the viewport minus the container padding, but that seems a bit janky when the container padding is defined in pixels.

In my testing, things seem to work fine without any of the new `max-width: _vw;` lines.

On very very small screen sizes, the content can overflow, but at that point the screen size doesn't seem realistic for any common device.

Reply:
We had put 94vw just because of the galaxy fold, which has a width of 280px, but we will make the necessary changes